prime-router/src/main/kotlin/Report.kt

```diff
@@ -604,25 +605,29 @@ class Report : Logging {
         }
     }
 
+    /**
+     * The second pass runs any mappers as needed for the given [toElement] and the [pass1Columns] data from the first
+     * pass.  Mappers will use data from either schema as inputs.
+     * @param mapping the mapping
+     * @return the mapped column
+     */
```

Comment:
In the future @whytheplatypus Theory world, can we get rid of this first pass / second pass glop, and replace with an ordered explicit list of mappers?  I hope so...